docs/user/pdf-version-support.md

```diff
@@ -9,7 +9,7 @@ PDF comes in the following versions:
 * 2001: 1.4
 * 2003: 1.5
 * 2004: 1.6
-* 2006 - 2012: 1.7, ISO 32000-1:2008
```

Comment:
Why dropping the ISO standard here?

Reply:
Not sure is needed, but fine to change to "2006: 1.7, ISO 32000-1:2008"